tests/native/aarch64cpu_asm_cache.py

```diff
@@ -0,0 +1,1762 @@
+assembly_cache = {
```

Comment:
```suggestion
# This is a cache of assembled instructions.
# This exists so that Manticore's tests can run without requiring that the
# Keystone dependency be installed.
# If additional test cases are added that require new instructions, this cache
# will need to be updated.
assembly_cache = {
```